haproxy/dockerdir/entrypoint.sh

```diff
@@ -23,4 +23,7 @@ if [ "$1" = 'haproxy' ]; then
     fi
     haproxy_opt+='-f /etc/haproxy/pxc/haproxy.cfg -p /etc/haproxy/pxc/haproxy.pid -S /etc/haproxy/pxc/haproxy-main.sock '
 fi
+
+test -e /opt/hookscript/hook.sh && source /opt/hookscript/hook.sh
```

Comment:
Please add 'percona' dir
```suggestion
test -e /opt/percona/hookscript/hook.sh && source /opt/percona/hookscript/hook.sh
```